### PR TITLE
SP-8125

### PR DESCRIPTION
**Pull request by eeerin**
add support for 3.10

### CODE REVIEW

File: .travis.yml
Comment (eeerin) [inline .travis.yml]: this isnt what I want though. People have asked for it but I'm not seeing any movement one here. https://travis-ci.community/t/add-python-3-10/12220 Waiting for it to actually get published before we merge this PR.

**Comment by eeerin** [comment]
their builds are failing https://app.travis-ci.com/github/hugovk/test/builds/243584339 for 3.10 and so the package didn't actually getting published.

**Review by spacetc62 — APPROVED** [review]

**Pull request merged**
